src/routes/legislative-process/+page.svelte

```diff
@@ -1,11 +1,45 @@
 <script land="ts">
-	import dayjs from 'dayjs';
-	import buddhistEra from 'dayjs/plugin/buddhistEra';
-	import 'dayjs/locale/th';
-	import Sidebar from '$components/LegislativeProcess/Sidebar.svelte';
+	import dayjs from "dayjs";
+	import buddhistEra from "dayjs/plugin/buddhistEra";
+	import "dayjs/locale/th";
+	import Sidebar from "$components/LegislativeProcess/Sidebar.svelte";
+	import DataTable from "../../components/DataTable/DataTable.svelte";
+
+	import "carbon-components-svelte/css/all.css";
 
 	dayjs.extend(buddhistEra);
-	dayjs.locale('th');
+	dayjs.locale("th");
+
+	const tableData = [
```

Comment:
`tableData` sounds too general. Maybe `proproserComparisonTableData` can help other people to understand what is this variable. Don't worry about the long name as long as it clear.